spec/views/users/index.html.erb_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'rails_helper'
 
-RSpec.describe 'users/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+user = FactoryBot.create(:user)
+
+RSpec.describe 'users/index.html.erb', type: :feature do
+  scenario 'display all usernames' do
+    visit '/'
+    expect(page).to have_content(user.name)
+  end
+
+  scenario 'display user profile picture' do
+    visit '/'
+    expect(page).to have_css("img[src='#{user.photo}']")
+  end
+
+  scenario 'display number of posts for each user' do
+    visit '/'
+    expect(page).to have_content(user.post_counter)
+  end
+
```

Comment:
- [ ] Kindly create more users and refactor you scenarios to demonstrate that multiple users appear on the page, instead of just one user.